textproc/lilypond-devel/Portfile

```diff
@@ -199,6 +205,13 @@ post-destroot {
 }
 
 variant docs description {Build documentation files} {
+    depends_build-append    port:ImageMagick \
```

Comment:
As explained in https://github.com/macports/macports-ports/pull/7040#issuecomment-626916421, this is not necessary currently.

Reply:
Removed.